README.md

```diff
@@ -19,7 +19,7 @@ builds configured to set the default target and sysroot for convenience.
 
 One could also use a standard Clang 8.0, build a sysroot from the sources
```

Comment:
Wait, this won't work with Clang 8.0, but will it work in 8.0.1?

Reply:
No, everything here works with Clang 8.0 too.

Reply:
If you're watching the LLVM commits, the only patches I landed for this were the ones adding tests for the "wasm32-wasi" triple. The support in the actual code was already present.

Reply:
Ah! I misunderstood that. Thank you for clarifying.